tools/import.go

```diff
@@ -0,0 +1,207 @@
+/*
```

Comment:
in Go you usually put executables in cmd/_name_/_name_.go It's kind of convention.

Reply:
done